components/create/CreateNft.vue

```diff
@@ -564,25 +539,6 @@ const autoTeleportActions = computed<AutoTeleportAction[]>(() => {
     },
   ]
 
-  if (needsListing.value) {
```

Comment:
this disables listing after minting for all no?

Reply:
It's only for Remark

<img width="562" alt="image" src="https://github.com/user-attachments/assets/942c0783-38db-4557-921a-7e023631b1d4">

Reply:
SO you won't be able to list nfts created on ahp/ ahk?

Reply:
We can still create nft and list it simultaneously on ahp/ahk. It was handled somewhere else.
The deleted part of codes in the PR is only for rmrk/ksm network.

<img width="775" alt="image" src="https://github.com/user-attachments/assets/b83144f6-6880-492b-880d-ae19d653c925">